service/handlers.go

```diff
@@ -25,24 +25,28 @@ func (rh *RouteHandler) ListWebhooks(w http.ResponseWriter, r *http.Request) (in
 	if err != nil {
 		return 500, err
 	}
-	objs, err := apiClient.GenericObject.List(&client.ListOpts{})
+	filters := make(map[string]interface{})
+	filters["kind"] = "webhookReceiver"
+	objs, err := apiClient.GenericObject.List(&client.ListOpts{
+		Filters: filters,
+	})
 	response := []model.Webhook{}
 	for _, obj := range objs.Data {
 		webhook, err := rh.convertToWebhookGenericObject(obj)
 		if err != nil {
-			logrus.Warnf("Skipping webhook %#v because: %v", obj, err)
+			logrus.Warnf("Skipping webhook #%s because: %v", obj.Id, err)
```

Comment:
Don't put `#` in front of the `%s`. I assume you are doing it because it is an Id, but it isnt actually a number and we don't use this `#` convention anywhere else.